chain/types/message.go

```diff
@@ -17,16 +17,16 @@ import (
 
 type Message struct {
 	// Address of the receiving actor.
-	To address.Address
+	To string
```

Comment:
These seem like a ton of unrelated changes. Not necessarily bad, but what's the motivation?

Reply:
Adding this info in the PR description.

Reply:
Thanks for the description.

This is a very unfortunate change on the whole. These structures are intended to match the on-chain structures exactly. This is particularly critical for future integration by other implementations, either through an RPC mechanism or CBOR files, where these are to be exchanged with another impl using essentially the same wire format as the network. I understand now that this change is driven by the inclusion of the `Message` inside the `MessageReceipt`, which is serialized as JSON into the expectation files. I didn't pick up on it the first time round, but altering that structure from its protocol definition is also undesirable for the same reason.

I think we need to find another way to do this. This format change will cripple the integration with other projects, and the ability to export actual chain CAR files using messages build with the builders here.